### PR TITLE
pds-api 68 : Add hits and took

### PR DESCRIPTION
**Pull request by al-niessner**
**Summary***
Added hits and took to the swagger. Adjusted the logic to fill in these items.

**Test Data and/or Report**

Using a standard curl, the results now show 'hits' and 'took':

```
$ curl --header 'Accept: application/json' "http://localhost:8080/bundles/urn:nasa:pds:izenberg_pdart14_meap::1.0/products?limit=1&start=0&fields=ops:Label_File_Info.ops:md5_checksum&only-summary=true"

{"summary":{

  "hits":65, "took":168,

"start":0,"limit":1,"sort":[],"properties":["vid","product_class","ops:Label_File_Info/ops:md5_checksum","ref_lid_instrument_host","ref_lid_investigation","lidvid","ref_lid_target","title","ref_lid_instrument","ops:Label_File_Info/ops:file_ref"]}}
```
Stating that there are 65 possibilities (hits) and it the processing time was 168 ms (took).

**Related Issues**

    * fixes issues in [pds-api] repo:
        - [68](https://github.com/NASA-PDS/pds-api/issues/68)
        - [87](https://github.com/nasa-pds/pds-api/issues/87)
        - [105](https://github.com/NASA-PDS/pds-api/issues/105)
        - #31 (actually this repo)

### CODE REVIEW

**Comment by al-niessner** [comment]
@jordanpadams @tloubrieu-jpl 

None of the checks run because cannot reliably update the api artifact for your automatic scripts to use. Anyway, ready for review as all 4 tickets are complete and tested as shown in the initial block of this pull request.

File: src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java [hunk 1]
```diff
@@ -181,6 +185,7 @@ private Products getCollectionChildren(String lidvid, int start, int limit, List
 
 	private Products getProductChildren(String lidvid, int start, int limit, List<String> fields, List<String> sort, boolean onlySummary) throws IOException
     {
+		long begin = System.currentTimeMillis();
     	if (!lidvid.contains("::")) lidvid = this.productBO.getLatestLidVidFromLid(lidvid);
     	MyBundlesApiController.log.info("request bundle lidvid, children of products: " + lidvid);
```
[the remark this answers is not in the document]
Comment (al-niessner) [reply src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java:201]: If you mean begin, then no. If you mean something else then I need you to be more explicit.

[the remark this answers is not in the document]
Comment (jordanpadams) [reply src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java:201]: @tloubrieu-jpl @al-niessner i poked through `git blame` and the revision history for this class and I can't quite seem to track down the change you may be referencing here, but I am also a newb to this codebase. that being said, it looks like this is the only blocker to this PR, so I am going to move forward and create a ticket for this change.

**Comment by jordanpadams** [comment]
@al-niessner would you mind quickly reviewing my changes [here](https://github.com/NASA-PDS/registry-api-service/pull/61/commits/8ac1a51dedd76cbd64042a5163bf552fa8e16d6d) (revert keyword removal) and [here](https://github.com/NASA-PDS/registry-api-service/pull/61/commits/f3c822030df76ea7279aa60689d6a5a74e1d1867) (fix merge conflicts) so we can merge the PR?

**Comment by al-niessner** [comment]
> @al-niessner would you mind quickly reviewing my changes [here](https://github.com/NASA-PDS/registry-api-service/pull/61/commits/8ac1a51dedd76cbd64042a5163bf552fa8e16d6d) (revert keyword removal) and [here](https://github.com/NASA-PDS/registry-api-service/pull/61/commits/f3c822030df76ea7279aa60689d6a5a74e1d1867) (fix merge conflicts) so we can merge the PR?

Keyword revert is fine if changes from main support it. Problem I had was that it only appeared in code but not in api maven artifact. If it is in api maven artifact, then it should be reinstated in the code.

The [second change](https://github.com/NASA-PDS/registry-api-service/pull/61/commits/f3c822030df76ea7279aa60689d6a5a74e1d1867) should not be returned to service. There is more comprehensive code for lid to lidvid lookups and the fix for returning 404 when lid/lidvid do not exist requires that those lines be removed. Looks like you may have a bad merge somewhere back on the main branch.

File: src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java [hunk 2]
```diff
@@ -99,7 +99,8 @@ public MyBundlesApiController(ObjectMapper objectMapper, HttpServletRequest requ
     private Products getCollectionChildren(String lidvid, int start, int limit, List<String> fields, List<String> sort, boolean onlySummary) throws IOException,LidVidNotFoundException
     {
     	long begin = System.currentTimeMillis();
-		if (!lidvid.contains("::") && !lidvid.endsWith(":")) lidvid = this.productBO.getLatestLidVidFromLid(lidvid);
+  		if (!lidvid.contains("::") && !lidvid.endsWith(":")) lidvid = this.productBO.getLatestLidVidFromLid(lidvid);
+
```
Comment (al-niessner) [inline src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java:103]: Not valid code anymore. The if part is not anyway.

File: src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java [hunk 3]
```diff
@@ -193,8 +194,9 @@ private Products getCollectionChildren(String lidvid, int start, int limit, List
 
 	private Products getProductChildren(String lidvid, int start, int limit, List<String> fields, List<String> sort, boolean onlySummary) throws IOException,LidVidNotFoundException
     {
-		long begin = System.currentTimeMillis();
+  		long begin = System.currentTimeMillis();
     	if (!lidvid.contains("::")) lidvid = this.productBO.getLatestLidVidFromLid(lidvid);
+
```
Comment (al-niessner) [inline src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java]: Not valid code anymore. The if part is not anyway.

File: src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java [hunk 4]
```diff
@@ -189,7 +194,9 @@ private Products getCollectionChildren(String lidvid, int start, int limit, List
 
 	private Products getProductChildren(String lidvid, int start, int limit, List<String> fields, List<String> sort, boolean onlySummary) throws IOException,LidVidNotFoundException
     {
-    	lidvid = this.productBO.getLatestLidVidFromLid(lidvid);
+  		long begin = System.currentTimeMillis();
+    	if (!lidvid.contains("::")) lidvid = this.productBO.getLatestLidVidFromLid(lidvid);
+
```
Comment (al-niessner) [inline src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java:199]: The if portion should be removed.

**Review by al-niessner — COMMENTED** [review]
The if part is no longer desired.

**Comment by al-niessner** [comment]
@jordanpadams @jimmie 

Fixed the fix. Should be good for a merge.

File: src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (tloubrieu-jpl) [reply src/main/java/gov/nasa/pds/api/engineering/controllers/MyBundlesApiController.java:199]: Sorry you can ignore this comment here. I miss placed it. I will recreate it where I was I intending to have it. Thanks

**Review by tloubrieu-jpl — APPROVED** [review]

**Pull request merged**